packages/grafana-eslint-rules/tests/no-untranslated-strings.test.js

```diff
@@ -437,8 +477,8 @@ const Foo = () => {
     },
 
     {
-      name: 'Cannot fix JSXExpression in attribute',
-      code: `const Foo = () => <div title={"foo"} />`,
+      name: 'Cannot fix JSXExpression in attribute if it is template literal',
+      code: `const Foo = () => <div title={\`foo\`} />`,
```

Comment:
A way round this limitation in future would be to prefer strings instead of template literals e.g. https://typescript-eslint.io/rules/no-unnecessary-template-expression